tests/test_runner.py

```diff
@@ -443,18 +443,6 @@ def test_run_wait(self, mock_jobsubmit, mock_getresultstree):
 
         self.assertEqual(result, 0)
 
-    @mock.patch('logging.warning')
-    @mock.patch('logging.error')
-    @mock.patch('skt.runner.BeakerRunner._BeakerRunner__jobsubmit')
-    def test_run_wait2(self, mock_logging, mock_logging_err, mock_jobsubmit):
-        """Ensure BeakerRunner.run works."""
-        # pylint: disable=W0613
-
-        # beaker_results.xml has test failures, returncode should be SKT_FAIL
-        result = misc.exec_on(self.myrunner, mock_jobsubmit,
-                              'beaker_results.xml', 1, 'Completed')
-        self.assertEqual(SKT_FAIL, result)
-
     @mock.patch('logging.warning')
```

Comment:
Why was this test removed? Is it about type and/or waiving?

Reply:
Because it was verifying the functionality with waiving=False, which is what was removed in the PR. Tests for waiving=True are still present